README.md

```diff
@@ -1,6 +1,9 @@
 # Texera
 Cloud-based text analytics using declarative and GUI-based workflows
 
+Check [this video](https://www.youtube.com/watch?v=67RY2S0pE-o) to see Texera in action!    
+To access the system, go to [texera.ics.uci.edu/twitter](http://texera.ics.uci.edu/twitter/) (currently accessible within UCI only)
```

Comment:
Change "To access the system" to "To access a live deployment on tweet analysis"